_data/organizers/organizing_committee.yml

```diff
@@ -0,0 +1,164 @@
+  twitter_url: https://twitter.com/sebineubauer
+  linkedin_url: https://www.linkedin.com/in/sebastian-neubauer-16626a79
+  github_url: https://github.com/sebastianneubauer
+  website_url: https://tech.blueyonder.com/
+  image: /assets/images/organizers/sebastian_neubauer.jpeg
+
+- name: Marco Bonzanini
+  desc: Marco is a Data Science consultant and trainer based in London (UK), organiser of PyData London
+  twitter_url: https://twitter.com/marcobonzanini
+  linkedin_url: https://www.linkedin.com/in/marcobonzanini/
+  github_url: https://github.com/bonzanini
+  website_url: https://marcobonzanini.com/
+  image: /assets/images/organizers/marco_bonzanini.jpeg
+
+- name: Motasim Rahman Kazmi
+  desc: Motasim is a CS undergrad and a full stack developer who loves data. He is an Intel Software Innovator and currently on the lookout for new and exciting opportunities. He also helps in organizing PyData Delhi meetups and conferences.
+  linkedin_url: https://www.linkedin.com/in/mrk333/
+  github_url: https://github.com/mrkazmi333
+  image: /assets/images/organizers/motasim_kazmi.jpg
+
+- name: Chetan Chawla
+  desc: Chetan is a data analyst and an astrophysics enthusiast. He builds stuff to solve real world problems, loves space and plays piano.
+  twitter_url: https://twitter.com/chetan_chawla_
+  linkedin_url: https://linkedin.com/in/chetan-chawla
+  github_url: https://github.com/chetanchawla
+  website_url: https://sites.google.com/view/chetanchawla
+  image: /assets/images/organizers/chetan_chawla.jpg
+
+- name: Sanket Verma
+  desc: Sanket works at Dunnhumby as a Research Data Scientist where he builds data science tools and is also Organizer of PyData Delhi. He loves to play the violin, computer games and sometimes think of saving the world!
+  twitter_url: https://twitter.com/msankeys963
+  linkedin_url: https://www.linkedin.com/in/msankeys963/
+  github_url: https://github.com/msankeys963
+  website_url: https://sanketverma.me/
+  image: /assets/images/organizers/sanket_verma.jpg
+
+- name: Matthijs Brouns
+  desc: Matthijs is the Data Science lead at Xccelerated where he trains junior Data Scientists. He joined the PyData Amsterdam committee as chair in 2019 
+  twitter_url: https://twitter.com/MatthijsBrs
+  linkedin_url: https://www.linkedin.com/in/mbrouns/
+  github_url: https://github.com/MBrouns
+  website_url: https://www.mbrouns.com/
+  image: /assets/images/organizers/matthijs.jpeg
+
+- name: Nancy Woo
+  desc: Nancy works at CVS Health, where she serves as a product manager and strategist to utilize data to shift people’s behaviors and make members healthier. In her free time, she enjoys traveling (not during a pandemic), learning, and watching sports.
+  linkedin_url: https://www.linkedin.com/in/nancyw19/
+  image: /assets/images/organizers/nancy_w.jpg
```

Comment:
Pictures for Nancy, Cheuk, Harshita and Matti seem to be broken because the image in assets has extension '.JPG' instead of '.jpg'.

Reply:
Will be fixed on next pull request.